docs/changes.rst

```diff
@@ -20,12 +20,10 @@ Change log
 ----------
 
 
-Version 1.6.0.dev1
+Version 1.6.0
 ^^^^^^^^^^^^^^^^^^
 
-This version contains all fixes up to version 1.5.x.
-
-Released: not yet
+Released: 2023-03-27
```

Comment:
Since version 1.5.1 was released and its fixes are in 1.6.0, and since the prior version listed in the change log is 1.5.0, we should give the user confidence about that, by having the following line:
```
This version contains all fixes up to version 1.5.1.
``

Reply:
Since the PR was merged, let's ignore this comment for this release.